tests/test_androidmodule_ctypes_finder.py

```diff
@@ -0,0 +1,124 @@
+
+import mock
+from mock import MagicMock
+import os
+import shutil
+import sys
+import tempfile
+
+sys.modules["jnius"] = MagicMock()  # fake jnius used later
```

Comment:
I promise this is my last wish, but could you remove this line and patch it explicitly when we need it. At at method level with the decorator, e.g. `@mock.patch.dict('sys.modules', jnius=MagicMock())` :pray: 
I really prefer when things are as local as possible